README.md

```diff
@@ -174,6 +174,9 @@ $ markdownlint-cli2 "**/*.md" "#node_modules"
 
 - The format of this file is a [CommonJS module][commonjs-module] that exports
   the object described above for `.markdownlint-cli2.jsonc`.
+- Instead of passing a `String` to identify the module name/path to load
+  for `customRules`, `markdownItPlugins`, and `outputFormatters`, the
```

Comment:
Super nit: Will "for" fit on the previous line? If so, will "corresponding" fit on this line?

Reply:
Yep.